modules/apps/static/portal-lpkg-deployer/portal-lpkg-deployer-impl/src/main/java/com/liferay/portal/lpkg/deployer/internal/BytesURLProtocolSupport.java

```diff
@@ -39,7 +39,7 @@
 /**
  * @author Shuyang Zhou
  */
-@Component(immediate = true, service = BytesURLProtocolSupport.class)
+@Component(service = BytesURLProtocolSupport.class)
```

Comment:
@slnn This class is not tracked or referenced or even used by anyone, but it registers a `URLStreamHandlerService` service when activating. I guess this service will be tracked by the OSGi framework, so we should keep this registration.

So I think we should keep this started by default.

Reply:
Please see https://github.com/liferay-core-infra/liferay-portal/pull/2838